base/intfuncs.jl

```diff
@@ -360,38 +360,39 @@ ndigits(x::Integer) = iszero(x) ? 1 : ndigits0z(x)
 ## ndigits with specified base ##
 
 # The suffix "nb" stands for "negative base"
-function ndigits0znb(n::Integer, b::Integer)
-    # precondition: b < -1 && !(typeof(n) <: Unsigned)
+function ndigits0znb(x::Integer, b::Integer)
```

Comment:
Only for consistency, most `ndigits` methods were receiving an `x`... Sorry to have done two things in this PR.